addons/website/static/src/xml/website.editor.xml

```diff
@@ -163,7 +163,7 @@
 
     <!-- Editor Toolbar -->
     <t t-extend="web_editor.toolbar">
-        <t t-jquery="#list .oe-toggle-checklist" t-operation="replace"/>
+        <t t-jquery="#list .oe-toggle-checklist" t-operation="after"/>
```

Comment:
What is done in 16.4 is an option was introduced called `showChecklist`, and the button is only added if the option is truthy.
What was done before your change was replacing the button with nothing in the context of `website`, so removing it.
What you do now is insert _nothing_ after the button. Not sure what that accomplishes.

Reply:
@Zynton my purpose for doing `t-operation="after"` was that when animate button for the floating toolbar was removed, the checklist it replaced by `t-operation="replace"` was never recovered which i found faulty, also when i checked in 16.4 the t-operation has been removed and position has been given to animate button. So i should simply give it position and it will work.

Reply:
I'm not sure I understand. You agree this change does nothing? `t-operation="after"` inserts the body of the `t` after the context node. But your `t` has no body (it's self-closing: `<t/>`). So it has nothing to insert.
Now I think you're saying you're going to use a different approach but I'm not sure, and if so I'm not sure which approach you're suggesting.

Reply:
@Zynton Updated the code, please have a look.